apis/v1alpha1/baselineadminnetworkpolicy_types.go

```diff
@@ -0,0 +1,156 @@
+/*
+Copyright 2022.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// All fields in this package are required unless Explicitly marked optional
+// +kubebuilder:validation:Required
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+//+kubebuilder:object:root=true
+//+kubebuilder:subresource:status
+
+// BaselineAdminNetworkPolicy is a cluster level resource that is part of the
+// adminNetworkPolicy API.
```

Comment:
```suggestion
// AdminNetworkPolicy API.
```

should we have another sentence or two both here and for `AdminNetworkPolicy`?

Reply:
Mirrored the sentence here for both BANP and ANP